utils/symbolicate-linux-fatal.py

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env python
+
+# TODOs:
+# * verbose output
+# * search symbols by name for the not <unavailable> ones
+
+import sys
+import lldb
```

Comment:
It'll pick up whatever is available in `PYTHONPATH`. If that's the standard (=non-Swift enabled) you're right we'll get as much as we get from `addr2line`. However, if the Swift-enabled one is available, we'll get symbols demangled etc. I'd hope longer term mainstream `lldb` will get Swift support. But `addr2line` probably won't.

Also, `lldb` approach _could_ be used on Darwin too (if there is some need for it).

What do you think @gribozavr ?